### PR TITLE
8340987: Open some TextArea awt tests 1

### PR DESCRIPTION
**Pull request by alisenchung**
First set of textarea awt tests to open during fall 2024 test sprint

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8340987](https://bugs.openjdk.org/browse/JDK-8340987): Open some TextArea awt tests 1 (**Bug** - P4)

### Reviewers
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**)
 * [Abhishek Kumar](https://openjdk.org/census#abhiscxk) (@kumarabhi006 - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/21435/head:pull/21435` \
`$ git checkout pull/21435`

Update a local copy of the PR: \
`$ git checkout pull/21435` \
`$ git pull https://git.openjdk.org/jdk.git pull/21435/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 21435`

View PR using the GUI difftool: \
`$ git pr show -t 21435`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/21435.diff">https://git.openjdk.org/jdk/pull/21435.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/21435#issuecomment-2403690388)

### CODE REVIEW

File: test/jdk/java/awt/TextArea/TextAreaAppendScrollTest2.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+import java.awt.BorderLayout;
+import java.awt.Frame;
+import java.awt.TextArea;
+
+/*
+ * @test
+ * @bug 6192116
+ * @summary Auto-scrolling does not work properly for TextArea when appending some text, on XToolkit
+ * @library /open/test/jdk/java/awt/regtesthelpers
```
Comment (kumarabhi006) [inline test/jdk/java/awt/TextArea/TextAreaAppendScrollTest2.java:32]: Please remove leading `/open/test/jdk` from all tests.

File: test/jdk/java/awt/TextArea/TextAreaAppendScrollTest2.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+ * @build PassFailJFrame
+ * @run main/manual TextAreaAppendScrollTest2
+ */
+
+public class TextAreaAppendScrollTest2 extends Frame {
```
Comment (kumarabhi006) [inline test/jdk/java/awt/TextArea/TextAreaAppendScrollTest2.java:37]: May not require to extent Frame.

File: test/jdk/java/awt/TextArea/TextAreaLimit.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        text = new TextArea();
+        add(text);
+        StringBuffer buf = new StringBuffer();
+        for (int i = 0; i < 40000; i++) {
+            buf.append(i + "\n");
+        }
+        text.setText(buf.toString());
+        text.setCaretPosition(buf.length());
+        text.requestFocus();
+        setSize(200,200);
```
Comment (kumarabhi006) [inline test/jdk/java/awt/TextArea/TextAreaLimit.java:68]:
```suggestion
        setSize(200, 200);
```

File: test/jdk/java/awt/TextArea/TextAreaAppendScrollTest2.java (hunk 2 above)
Comment (alisenchung) [reply test/jdk/java/awt/TextArea/TextAreaAppendScrollTest2.java:37]: will leave as is to match original test

**Review by prrace — APPROVED** [review]

**Review by kumarabhi006 — APPROVED** [review]

**Pull request closed without merging**